crates/bevy_utils/Cargo.toml

```diff
@@ -10,23 +10,18 @@ keywords = ["bevy"]
 
 [features]
 default = ["std", "serde"]
-std = [
-  "alloc",
-  "tracing/std",
-  "ahash/std",
-  "dep:thread_local",
-  "ahash/runtime-rng",
-]
-alloc = ["hashbrown/default"]
+std = ["alloc", "tracing/std", "foldhash/std", "dep:thread_local"]
+alloc = ["hashbrown"]
 detailed_trace = []
 serde = ["hashbrown/serde"]
 
 [dependencies]
-ahash = { version = "0.8.7", default-features = false, features = [
-  "compile-time-rng",
-] }
+foldhash = { version = "0.1.3", default-features = false }
 tracing = { version = "0.1", default-features = false }
-hashbrown = { version = "0.14.2", default-features = false }
+hashbrown = { version = "0.15.0", features = [
+  "equivalent",
+  "raw-entry",
+], optional = true, default-features = false }
```

Comment:
This is to explicitly exclude `default-hasher` here. It gives us the flexibility to change our hasher whenever since nobody will be relying on the default hasher, even though it *is* a pain to do this.

Reply:
FYI `0.15.0` was yanked so you either want to use `0.15.1` (or `0.15` maybe?).

Reply:
I actually marked it as 0.15.1 here because 0.15.0 was un-yanked, but may have some advisories filed for it, even though none of them affect us. Don't want to accidentally trigger any lints.